README.md

```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
 
 ![](stark.gif)
 
+-------------------------------------------------------------------------------
+
+## Objective
+
+The objective of Maestro is to allow a beginner to use services like 
+[Twilio](https://www.twilio.com/products) without being slowed by the details
+of how to implement the API. This will help beginners build more/better 
+applications which will help encourage them to continue learning.
+
+## Usage
+
+The JS file containing the library is found at `examplehost.com/static/baton.js`
+
+Once that file is executed, the global variable `maestro` which contains all of
+the module libraries will be ready. Any messages from the server that are not
+caught with callbacks will automatically be logged in the console.
+
+Open up the javascript console and [try it out!](http://maestro.ngrok.com/static/)
+
+### Examples
+
+```
+maestro.Twilio.recieveSms("*",function(e){ //recieve any incoming sms
+  maestro.Twilio.makeCall(e.From,e.To,maestro.Twilio.twiml().play(e.Body)); //call back  playing any sound file in the message
+  console.log("Calling",e.From,"from",e.To,"playing",e.Body)
+});
+```
```

Comment:
Perhaps this code would be easier to read if function arguments where separated by spaces?